### DISCUSSION TITLE
platform/actors/development/actor-definition/output-schema

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1151] [Comments]
# platform/actors/development/actor-definition/output-schema

Learn how to define and present your output schema in an user-friendly output UI.

https://docs.apify.com/platform/actors/development/actor-definition/output-schema

**Comment by max4ever** [comment]
Is it possible to specify that the output should read from a dataset with a specific name? In case the actor writes only to a named dataset?

**Reply by surajswaraj88-dotcom** [reply]
{
    "actorOutputSchemaVersion": 1,

    "title": "Cypress test report output",
    "description": "Test report from Cypress",
    "type": "object",

    "properties": {
        "reportUrl": {
            "type": "string",
            "title": "HTML Report",
            "template": "{{links.apiDefaultKeyValueStoreUrl}}/records/report.html"
        }
    }
}
